hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/metrics/HybridSubscriptionMetricsRepository.java

```diff
@@ -43,8 +52,15 @@ public HybridSubscriptionMetricsRepository(GraphiteClient graphiteClient, Metric
     @Override
     public SubscriptionMetrics loadMetrics(TopicName topicName, String subscriptionName) {
         String rateMetric = metricPath(topicName, subscriptionName);
+        String timeouts = metricPathTimeouts(topicName, subscriptionName);
+        String otherErrors = metricPathOtherErrors(topicName, subscriptionName);
 
-        GraphiteMetrics metrics = graphiteClient.readMetrics(rateMetric);
+        List<String> metricsPathsForHttpStatuses = Arrays.stream(HTTP_STATUS_CODE_FAMILIES_FOR_METRICS)
```

Comment:
You created a list of statuses, but actually treat it more like a map than a collection to iterate from. These metricsPathsForHttpStatuses.get(0) are not explicit and error-prone.

I suggest replacing HTTP_STATUS_CODE_FAMILIES_FOR_METRICS array and operating on it with 3 explicit variables containing the metricsStatusPaths for 2xx, 4xx and 5xx codes.